### ISSUE TITLE
Can the filters exclude certain hashtags?

### ISSUE THREAD

**Issue by zstadler** [OSMCha/osmcha-frontend#755]
## *Brief Description*

Is it possible to filter-out changesets with a given metadata?
For example, I would like to **exclude** changesets that have a `hashtags` that contains `#hotosm-project`.

## What is the motivation / use case for this feature?

When following mass changes or deletions in my area, I would like to avoid looking at the volume of changesets related to organized mapping efforts, such as HOT. I'm assuming that they have their own quality assurance process.

## What  is the current behaviour, (attach relevant screenshots) ?

I see a reference to `hashtags` in the hints for the `Other metadata` filter:

![image](https://github.com/user-attachments/assets/ad8934c5-d188-495b-937c-39e67e9030b2)

It is not clear to me if the negation of `hashtags=#hotosm-project` is available in this filter, or elsewhere

**Comment by jake-low** [comment]
I don't believe that it's currently possible to negate metadata filters (or any other filters). I think this is a good feature request though.

**Comment by martien-176** [comment]
I strongly support this feature request. Negation would imo expand filter possibilities greatly.

And not only in the Comment filter, also eg "OSM features tags", Source etc.
